Current/Performance counters permission check.sql

```diff
@@ -0,0 +1,31 @@
+IF EXISTS (SELECT 1 FROM sys.objects WHERE object_id = OBJECT_ID('[SQLCop].[DmOsPerformanceCountersPermissionErrors]') AND TYPE = 'FN')
+DROP FUNCTION [SQLCop].[DmOsPerformanceCountersPermissionErrors]
+GO
+
+CREATE FUNCTION [SQLCop].[DmOsPerformanceCountersPermissionErrors]()
+RETURNS VARCHAR(MAX)
+AS
+BEGIN
+    DECLARE @Result VARCHAR(MAX)
+    SET @Result = ''
+
+    IF (SERVERPROPERTY('EngineEdition') = 5)
```

Comment:
See managed instance comment above

Reply:
I've done some testing and the permissions required on a managed instance match regular SQL Server so I think this function is ready to go.